src/pages/audio-rewards-page/store/sagas.ts

```diff
@@ -96,12 +100,21 @@ function* claimChallengeRewardAsync(
   const quorumSize = remoteConfigInstance.getRemoteVar(
     IntKeys.ATTESTATION_QUORUM_SIZE
   )
-  const oracleEthAddress = remoteConfigInstance.getRemoteVar(
+  let oracleEthAddress = remoteConfigInstance.getRemoteVar(
     StringKeys.ORACLE_ETH_ADDRESS
   )
-  const AAOEndpoint = remoteConfigInstance.getRemoteVar(
+  let AAOEndpoint = remoteConfigInstance.getRemoteVar(
     StringKeys.ORACLE_ENDPOINT
   )
+  if (ENVIRONMENT === 'development') {
+    const oracleEthAddresses = (REACT_APP_ORACLE_ETH_ADDRESSES || '').split(',')
+    if (oracleEthAddresses.length > 0) {
+      oracleEthAddress = oracleEthAddresses[0]
+    }
+    if (REACT_APP_AAO_ENDPOINT) {
+      AAOEndpoint = REACT_APP_AAO_ENDPOINT
+    }
+  }
```

Comment:
Could probably pull this whole bit into a function

Reply:
let's do it